frontend/src/stimulus/controllers/dynamic/work-packages/relations-tab/scroll.controller.ts

```diff
@@ -0,0 +1,18 @@
+import { Controller } from '@hotwired/stimulus';
+
+export default class ScrollController extends Controller {
+  static values = { targetId: String };
+  declare targetIdValue:string;
+
+  connect() {
+    // Wait to ensure DOM is fully loaded
+    setTimeout(() => {
```

Comment:
`setTimeout` does not automatically ensure you that the DOM is fully loaded afaik, there might still be some delays that could prevent that could from being executed.